pkg/blob/controllerserver.go

```diff
@@ -107,6 +108,10 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 			secretName = v
 		case secretNamespaceField:
 			secretNamespace = v
+		case isCacheEnabledField:
```

Comment:
I think it's better add a new mount type, we already have `fuse`, `nfs`, this is the third mount type: `edgecache`
https://github.com/kubernetes-sigs/blob-csi-driver/blob/02ba84f85e6ac1ff153a7648297a91425e1db7ed/pkg/blob/blob.go#L118

Reply:
+1 on this suggestion, and +10 if it eliminates the need for the special suffixes on edge cache mount paths.

Reply:
The latest update removes this param and uses a new protocol option "edgecache"

See change here: https://github.com/avoltz/blob-csi-driver/pull/2/commits/e7c21ba6d2aceabbbfefdf09f6bb38f896c7cb00